src/checksum/arm64/crc32-arm64.c

```diff
@@ -59,30 +59,45 @@ static uint32_t
 __wt_checksum_with_seed_hw(uint32_t seed, const void *chunk, size_t len)
 {
     uint32_t crc;
-    size_t nqwords;
-    const uint8_t *p;
     const uint64_t *p64;
+    const uint32_t *p32;
+    const uint16_t *p16;
+    const uint8_t *p8;
```

Comment:
Personally I find the code more readable using a single `const uint8_t*` pointer and making each access specify a width, and each increment increment in bytes so that it matches the `len` decrement. But I know that is a matter of taste, so I'll leave that up to your discretion.